import_specifications/schema/dts_manifest.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
I don't know the structure of the DTS manifest so I'm going to have to trust this is correct and complete